.pre-commit-config.yaml

```diff
@@ -94,3 +94,9 @@ repos:
         args:
           - -w
           - -s
+  - repo: https://github.com/compilerla/conventional-pre-commit
+    rev: v3.4.0
+    hooks:
+      - id: conventional-pre-commit
+        stages: [commit-msg]
+        args: []
```

Comment:
If args is empty, do we need it?

Reply:
Probably not, I just followed their docs outline [here](https://github.com/compilerla/conventional-pre-commit)